jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Slider.java

```diff
@@ -109,15 +109,14 @@ public static int getInteger(String attr, WebElement el, int defaultValue) {
   }
 
   private String setNewStyle(String style, int value) {
-    style = style.replaceAll("[-?0-9]+", "");
-    int perInx = style.indexOf("%");
-    return style.substring(0, perInx) + 0 + style.substring(perInx, style.length()-2)
-        + value + style.substring(style.length()-2);
+    String styleLocal = style.replaceAll("[-?0-9]+", "");
+    int perInx = styleLocal.indexOf("%");
+    return styleLocal.substring(0, perInx) + 0 + styleLocal.substring(perInx, style.length() - 2)
+            + value + styleLocal.substring(style.length() - 2);
```

Comment:
One of the tests is failing due to this update.
Update style.length() to styleLocal.length()